src/main/kotlin/no/nav/syfo/dialogmelding/DialogmeldingService.kt

```diff
@@ -12,15 +12,14 @@ class DialogmeldingService(private val behandlerVarselService: BehandlerVarselSe
 
     fun handleDialogmelding(dialogmeldingDTO: KafkaDialogmeldingDTO) {
         if (dialogmeldingDTO.msgType == DIALOGMELDING_TYPE_SVAR) {
-            val dialogmeldingSvar = dialogmeldingDTO.toDialogmeldingSvar()
-            dialogmeldingSvar.innkallingDialogmoteSvar?.let { innkallingSvar ->
+            dialogmeldingDTO.toDialogmeldingSvar().find { dialogmeldingSvar ->
```

Comment:
Foreslår å bytte til  noe ala toDialogmeldingSvarAlternativer el toDialogmeldingSvarAlternatives, så det blir lettere å skjønne at vi gjetter litt.

Reply:
Ta gjerne en navneendring eller to for å gjøre det mer tydelig hva som gjøres i denne endringen. Tok litt tid å skjønne hva man gjorde her.

Reply:
Forsøkt med noen navnetilpasninger nå.